diracx-routers/src/diracx/routers/__init__.py

```diff
@@ -440,3 +446,46 @@ async def db_transaction(db: T2) -> AsyncGenerator[T2]:
         if reason := await is_db_unavailable(db):
             raise DBUnavailable(reason)
         yield db
+
+
+class ClientMinVersionCheckMiddleware(BaseHTTPMiddleware):
+    """Custom FastAPI middleware to verify that
+    the client has the required minimum version.
+    """
+
+    def __init__(self, app: FastAPI):
+        super().__init__(app)
+        self.min_client_version = get_min_client_version()
```

Comment:
Based on `Note2` in my previous comment:

```suggestion
        self.min_client_version = get_min_client_version()
        self.parsed_min_client_version = parse(self.min_client_version)
```

Reply:
Here I guess you don't need to catch it because it will be displayed on server side for the operators that misconfigured DiracX. What do you think?

Reply:
Yes, I agree!